gemet/thesaurus/edit_views.py

```diff
@@ -525,6 +533,13 @@ def get_context_data(self, **kwargs):
 
         for concept in old_concepts:
             concept.set_attributes(self.langcode, ['prefLabel'])
+            concept.url = reverse(
+                EDIT_URL_NAMES[concept.namespace.heading],
```

Comment:
Fetch namespaces at the same time with concepts by using `select_related`. This way you won't issue a new query for each concept.